Documentation/benchmarks/etcd-3-watch-memory-benchmark.md

```diff
@@ -0,0 +1,72 @@
+# Watch Memory Usage Benchmark
+
+*NOTE*: The watch features are still being actively developed and we expect the memory usage to change over time. However we do expect that memory usage should not be significantly increased from the figures below.
```

Comment:
The honest truth is, I don't know that the "Note" here serves any real purpose. It says almost nothing. If we mean "We currently eat memory for 5 meals a day, but will go on a diet in future development." then we should say that.